books/free-programming-books-hi.md

```diff
@@ -1,10 +1,16 @@
 ### Index
 
 * [C++](#cpp)
+* [JAVA](#java)
 
 
 ### <a id="cpp"></a>C++
 
 * [C++ Brief Notes \| Hindi](https://ehindistudy.com/2020/12/01/cpp-notes-in-hindi/) - Yugal Joshi
 * [C++ in Hindi \| Hindi](https://www.bccfalna.com/IOC-AllEBooks/CPPinHindi.pdf) - Kuldeep Chand (PDF)
 * [C++ Introduction Book \| Hindi](https://ncsmindia.com/wp-content/uploads/2012/04/c++-hindi.pdf) - NCMS India (PDF)
+
+### <a id="java"></a>java
```

Comment:
PDF drive links are not supported. Please provide links to authorized distributions